docs/tutorial/en/source/tutorial/MCP.py

```diff
@@ -0,0 +1,149 @@
+# -*- coding: utf-8 -*-
+"""
+.. _MCP:
+
+MCP
+====================
+
+In this tutorial, we will briefly introduce what MCP is and show how to use MCP in AgentScope. We will also demonstrate how to configure an existing AgentScope multi-agent application as an MCP server. Finally, to help advanced developers gain a deeper understanding, we will discuss how MCP integrates with AgentScope.
+
+.. note:: 1. Before reading this chapter, please read the "Tools" chapter to understand the basic usage of the `ServiceToolkit` module.
+
+ 2. Using MCP in AgentScope requires your Python version to be at least 3.10, and you must have installed the `agentscope[service]` version.
```

Comment:
Not sure if this two notes will be displayed normally if there is a blank line between them